src/assets/i18n/fr.json

```diff
@@ -2,645 +2,644 @@
-        "delete-all-non-visible-elements-tooltip": "Delete all non-visible elements, such as nodes, trainruns and notes",
-        "delete-all-non-visible-elements": "Delete all non-visible elements",
-        "visible-elements": "Visible elements",
-        "delete-all-visible-trainruns-tooltip": "Delete all visible trainruns",
-        "delete-all-visible-trainruns": "Delete all visible trainruns",
-        "delete-all-visible-notes-tooltip": "Delete all visible notes",
-        "delete-all-visible-notes": "Delete all visible notes",
-        "delete-all-visible-elements-tooltip": "Delete all visible elements, such as nodes, trainruns and notes",
-        "delete-all-visible-elements": "Delete all visible elements",
-        "merge-netzgrafik-title": "Netzgrafik - Merge",
-        "add-netzgrafik-as-copy-tooltip": "Add netzgrafik as a copy (trainruns, nodes, notes)",
-        "add-netzgrafik-as-copy": "Add netzgrafik as a copy",
-        "merge-netzgrafik-tooltip": "Merge netzgrafik (trainruns, nodes, notes)",
-        "merge-netzgrafik": "Merge netzgrafik",
-        "delete": "Delete",
-        "on-clear-delete-all-non-visible-elements": "Should all non-visible elements be permanently deleted from the netzgrafik?",
-        "on-clear-delete-all-visible-elements": "Should all visible elements be permanently deleted from the netzgrafik?",
-        "on-clear-delete-all-visible-trainruns": "Should all visible trainruns be permanently deleted from the netzgrafik?",
-        "on-clear-delete-all-visible-notes": "Should all visible notes be permanently deleted from the netzgrafik?",
+        "delete-netzgrafik-title": "Netzgrafik - Supprimer",
+        "non-visible-elements": "Éléments non visibles",
+        "delete-all-non-visible-elements-tooltip": "Supprimer tous les éléments non visibles, tels que les noeuds, les lignes de train et les notes",
+        "delete-all-non-visible-elements": "Supprimer tous les éléments non visibles",
+        "visible-elements": "Éléments visibles",
+        "delete-all-visible-trainruns-tooltip": "Supprimer toutes les lignes de trains visibles",
+        "delete-all-visible-trainruns": "Supprimer toutes les lignes de trains visibles",
+        "delete-all-visible-notes-tooltip": "Supprimer toutes les notes visibles",
+        "delete-all-visible-notes": "Supprimer toutes les notes visibles",
+        "delete-all-visible-elements-tooltip": "Supprimer tous les éléments visibles, tels que les noeuds, les lignes de train et les notes",
+        "delete-all-visible-elements": "Supprimer tous les éléments visibles",
+        "merge-netzgrafik-title": "Netzgrafik - Fusionner",
+        "add-netzgrafik-as-copy-tooltip": "Ajouter un netzgrafik en tant que copie (lignes de train, noeuds, notes)",
+        "add-netzgrafik-as-copy": "Ajouter un netzgrafik en tant que copie",
+        "merge-netzgrafik-tooltip": "Fusionner les réticulaires (lignes de train, noeuds, notes)",
```

Comment:
réticulaires ?

Reply:
"Réticulaire" means network graphic in french, as "Graphique de réseau" has a quite large range of meaning. It is actually the word used by the business people, at least at SNCF Réseau:
https://www.sncf-reseau.com/medias-publics/2023-11/R%C3%A9ticulaire%20PER%20Marseille-Vintimille%20SA%202025%20-%2030%20Novembre%202023.pdf

I'll put that in the open-discussion list